indy_client/test/cli/test_pool_upgrade_schedule.py

```diff
@@ -1,27 +0,0 @@
-import dateutil
-import pytest
-from datetime import timedelta, datetime
-
-from indy_client.test.cli.test_pool_upgrade import poolUpgradeSubmitted
-from indy_client.test.cli.test_pool_upgrade import poolUpgradeScheduled
-from indy_node.test.upgrade.conftest import validUpgrade as _validUpgrade
-
-
-@pytest.fixture(scope='function')
```

Comment:
This tests need to be migrated to SDK and moved to indy-node folder

Reply:
I think we have this test case duplicated in
indy-node/indy_node/test/upgrade
Please, tell me, if we need to integrate this in node